lib/push.js

```diff
@@ -78,7 +79,17 @@ var reasonToEvents = {
   }
 }
 
-module.exports = function (log, db) {
+module.exports = function (log, db, config) {
+  var vapid
+  if (config.vapidKeysFile) {
+    var vapidKeys = JSON.parse(fs.readFileSync(config.vapidKeysFile))
```

Comment:
You make a good point about conflating code and data, I've never considered it from that perspective before. The benefit I see is in making potential future changes to the codebase easier, either changing code modules to be dumb data files or vice versa.